packages/codebytes/src/MonacoEditor/theme.ts

```diff
@@ -58,6 +58,7 @@ const theme = ({
     'editorIndentGuide.background': ui.indent.inactive,
     'editorIndentGuide.activeBackground': ui.indent.active,
     'editorWhitespace.foreground': syntax.comment,
+    'editorLineNumber.foreground': '#9FA2AC',
```

Comment:
i know not ideal, but this solved axe color contrast issues, and since it sounds like web plat is working on a better shared monaco anyway, this can be okay temporarily?

Reply:
Maybe we should add this to the [ui colors const in the colorsDark.ts file](https://github.com/Codecademy/client-modules/blob/main/packages/codebytes/src/MonacoEditor/colorsDark.ts#L43-L50). Something like `lineNumber: theme.colors['navy-400']` (what #9FA2AC lines up with in [swatches](https://gamut.codecademy.com/?path=/docs/foundations-theme--colors#swatches)) and then we could reference it as `ui.lineNumber` here.